nx/include/switch/services/hid.h

```diff
@@ -600,6 +600,18 @@ typedef struct HidVibrationValue
     float freq_high; ///< High Band frequency in Hz.
 } HidVibrationValue;
 
+static inline u32 _hidControllerIDToOfficial(HidControllerID id) {
```

Comment:
Remove _ prefix.

Reply:
This function was moved; and it originally had the underscore prefix...